iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -54,10 +56,46 @@ public double getMaxQ() {
 
     private final TreeMap<Double, Point> points;
 
-    ReactiveCapabilityCurveImpl(TreeMap<Double, Point> points) {
+    private static void checkPointsSize(TreeMap<Double, Point> points) {
         if (points.size() < 2) {
-            throw new IllegalStateException("Points size must be >= 2");
+            throw new IllegalStateException("points size should be >= 2");
+        }
```

Comment:
Again. This check should be done in the adder.
And it can add a regression.
Maybe if there is only one point, you should use the vertical extrapolation as before.

Reply:
Oups I didn't see it existed before and this is just a refactoring. Strange it is not in the adder.